website/docs/tutorials/concat_bert.md

```diff
@@ -0,0 +1,262 @@
+
+```yaml
+
+model_config:
+  concat_bert:
+    # Either pretraining or classification
+    bert_model_name: bert-base-uncased
+    direct_features_input: false
+    # Dimension of the embedding finally returned by the modal encoder
+    modal_hidden_size: 2048
+    # Dimension of the embedding finally returned by the text encoder
+    text_hidden_size: 768
+    # Used when classification head is activated
+    num_labels: 2
+    # Number of features extracted out per image
+    num_features: 1
+
+    image_encoder:
+      type: resnet152
+      params:
+        pretrained: true
+        pool_type: avg
+        num_output_features: 1
+
+    text_encoder:
+      type: transformer
+      params:
+        bert_model_name: bert-base-uncased
+        hidden_size: 768
+        num_hidden_layers: 12
+        num_attention_heads: 12
+        output_attentions: false
+        output_hidden_states: false
+
+    classifier:
+      type: mlp
+      params:
+        # 2048 + 768 in case of features
+        # Modal_Dim * Number of embeddings + Text Dim
+        in_dim: 2816
+        out_dim: 2
+        hidden_dim: 768
+        num_layers: 2
+
+```
+
+## Experiment Config
+
+In the next step, we will create the experiment config which will tell MMF which dataset, optimizer, scheduler, metrics for evalauation to use. We will save this config in `configs/experiments/concat_bert/defaults.yaml`:
+
+```yaml
+
+includes:
+- configs/datasets/hateful_memes/bert.yaml
+
+model_config:
+  trying:
+    classifier:
+      type: mlp
+      params:
+        num_layers: 2
+    losses:
+    - type: cross_entropy
+
+scheduler:
+  type: warmup_linear
+  params:
+    num_warmup_steps: 2000
+    num_training_steps: ${training.max_updates}
+
+optimizer:
+  type: adam_w
+  params:
+    lr: 5e-5
+    eps: 1e-8
+
+evaluation:
+  metrics:
+  - accuracy
+  - binary_f1
+  - roc_auc
+
+training:
+  batch_size: 64
+  lr_scheduler: true
+  max_updates: 22000
+  early_stop:
+    criteria: hateful_memes/roc_auc
+    minimize: false
+
+```
+
+We include the `bert.yaml` config in this as we want to use BERT tokenizer for preprocessing our language data. With both the configs ready we are ready to launch training and evaluation using our model on Hateful Memes dataset. You can read more about the MMF’s configuration system [here](https://mmf.sh/docs/notes/configuration).
+
+
+
+## Training and Evaluation
+
+Now we are ready to train and evaluate our model with the experiment config we created in previous step.
+
+```bash
+mmf_run config="configs/experiments/concat_bert/defaults.yaml" \
+    model=concat_bert \
+    dataset=hateful_memes \
+    run_type=train_val
+```
+
+When training ends it will save a final model `concat_bert_final.pth` in the experiment folder under `./save` directory. The command will also generate validation scores after the training gets over.
```

Comment:
Add link to general checkpointing tutorial here about prediction and evaluation.